frameworks/moveos-stdlib/sources/object.move

```diff
@@ -494,6 +494,11 @@ module moveos_std::object {
         native_object_size(obj.id)
     }
 
+    /// List all field names of the object
+    fun list_field_keys<T: key>(obj: &Object<T>): vector<address> {
```

Comment:
Should we support pagination? if the object has too many fields, this call will out_of_gas